src/System.Diagnostics.Process/src/Resources/Strings.resx

```diff
@@ -315,4 +315,7 @@
   <data name="UserDoesNotExist" xml:space="preserve">
     <value>User with name '{0}' was not found.</value>
   </data>
+  <data name="ArgumentAndArgumentListInitialized" xml:space="preserve">
+    <value>The caller may either use Arguments or ArgumentList.</value>
```

Comment:
Consider augmenting or changing it to clarify that both can't be used, e.g.
"Only one of Arguments or ArgumentList may be used."